02.extensions/zoocart/01.basics/04.settings/_elements/default.md

```diff
@@ -44,7 +44,7 @@ When adding the Element to a Type, the following configuration is available unde
 
 | Field       | Description |
 | :---------- | :---------- |
-| Elements | The list of Elements that would be possible to override by each Variation. |
+| Elements | The list of Elements that would be possible to override by each Variation. Not compatible with image galeries and media.|
```

Comment:
Change it for "Not compatible with JS dependent elements, like Image Gallery or Media."